docs/get-started.md

```diff
@@ -0,0 +1,167 @@
+# Instrument a Go application or services automatically
+
+You can instrument a Go executable using OpenTelemetry without having
+to write additional code. All you need to do is configure a few environment
+variables and run the instrumentation with elevated privileges.
+
+The following example shows how to instrument a simple Go application
+automatically on a Linux host, through Docker, and using Kubernetes.
+
+## Prerequisites
+
+To instrument an application automatically, you need the following:
+
+- Binary file or Docker image of OpenTelemetry Go instrumentation
+- Go 1.17 and higher in your system
+- Linux with kernel version 4.4 and higher
```

Comment:
eBPF also needs to be loaded on the system.

Reply:
Oh, I though it was on by default.

Reply:
@pellared How do you enable it? Is this only for kernels of the 4.x family?

Reply:
I think it should be enabled by default 🤷

Reply:
@edeNFed any insights?

Reply:
Yeah, looking more at this, the default compilation will have eBPF enabled. I think this can be resolved.